src/components/Card/Card.styled.jsx

```diff
@@ -45,7 +46,8 @@ const CardDescription = styled.p`
   font-weight: 400;
   line-height: 1.43;
   letter-spacing: 0.01071em;
-  color: rgba(0, 0, 0, 0.54);
+  color: ${(props) =>
+    props.theme === 'light' ? 'rgba(0, 0, 0, 0.54)' : 'rgba(255, 255, 255, 0.7)'};
```

Comment:
A cleaner and less redundant way to implement theme is by handle all your color variables inside a theme and just toggle in the root of your tree the theme. It will automatically change colors according to the variables:  https://www.smashingmagazine.com/2020/04/dark-mode-react-apps-styled-components/